src/libs/tools/src/plugindatabase.cpp

```diff
@@ -223,6 +223,21 @@ std::string ModulesPluginDatabase::lookupInfo (PluginSpec const & spec, std::str
 	return plugin->lookupInfo (which);
 }
 
+PluginDatabase::func_t ModulesPluginDatabase::getSymbol (PluginSpec const & spec, std::string const & which) const
+{
+	PluginPtr plugin;
+	try
+	{
+		plugin = impl->modules.load (spec.getName (), spec.getConfig ());
+	}
+	catch (...)
+	{
+		throw;
+	}
```

Comment:
Is it really necessary to catch and re-throw the same error? If you are not going to clean up something else in the catch-clause, it seems not necessary.